multisig/multisig.go

```diff
@@ -0,0 +1,43 @@
+package multisig
+
+import (
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/pairing/bn256"
+	"go.dedis.ch/kyber/v3/sign/bls"
+	"go.dedis.ch/kyber/v3/util/random"
+)
+
+type (
+	// PrivateKey is a generic type for private key.
+	PrivateKey = kyber.Scalar
+	// PublicKey is a generic type for public key.
+	PublicKey = kyber.Point
+)
+
+var suite = bn256.NewSuite()
+
+// GenerateKey returns random key pair.
+func GenerateKey() (PrivateKey, PublicKey) {
+	return bls.NewKeyPair(suite, random.New())
+}
+
+// Sign returns message signature.
+func Sign(key PrivateKey, msg []byte) ([]byte, error) {
+	return bls.Sign(suite, key, msg)
+}
+
+// Verify checks if sig is msg signature.
+func Verify(pub PublicKey, msg, sig []byte) error {
```

Comment:
SignRFC6979 / Verify has different function signature.. mb we can change signature here, what you think?